src/main/kotlin/com/sourcegraph/cody/edit/HistoryManager.kt

```diff
@@ -1,49 +0,0 @@
-package com.sourcegraph.cody.edit
-
-/** Manages user prompt history in memory for a session. */
-class HistoryManager<T>(private val capacity: Int) {
```

Comment:
removed in favor of [TextAreaHistoryManager.kt](https://github.com/sourcegraph/jetbrains/pull/1786/files#diff-7ee152a571f6e3a8ba8a5f2cf1d1e9642da2ed4b8c1164601fef2ba01a020e74)